src/runtime/engines/singularity/container.go

```diff
@@ -506,7 +506,7 @@ func (c *container) addRootfsMount(system *mount.System) error {
 			if err = ecl.ValidateConfig(); err != nil {
```

Comment:
Just one point we already discussed about moving ECL validation block in `prepare.go`